dags/common/cleanup.py

```diff
@@ -1,11 +1,19 @@
 def clean_whitespace_characters(input):
     return " ".join(input.split())
 
+
 def convert_html_subsripts_to_latex(input):
     from re import sub
 
     input = sub("<sub>(.*?)</sub>", r"$_{\1}$", input)
     input = sub("<inf>(.*?)</inf>", r"$_{\1}$", input)
-    input = sub("<sup>(.*?)</sup>", r"$^{\1}$", input)   
+    input = sub("<sup>(.*?)</sup>", r"$^{\1}$", input)
 
     return input
+
+
+def remove_specific_tags(value, keep=[]):
+    from bleach.sanitizer import Cleaner
+
+    cleaner = Cleaner(tags=keep, strip=True)
+    return cleaner.clean(value).replace("\n", "")
```

Comment:
```suggestion
    from bleach import clean
    return clean(value, strip=True, tags=keep)
```

Do we need to remove the new lines? Was it the case before?

Reply:
yes, because strip adds new lines, instead if the tag, which was inside of the allowed tag.
For example, a is allowed:
input:
``` <a><b>'example'</b></a>```
output: 
```
<a>
'example'
</a>
```
No, it wasn't a case before

Reply:
@ErnestaP thank you so much for the explanation! Testing this case it doesn't seem to be the case:

```python
In [1]: from bleach import clean                                                                                                                                                                                                                                                                                                                                   
In [2]: test = "<a><b>'example'</b></a>"                                                                                                                                                                                                                                                                                                                           
In [3]: clean(test, strip=True, tags=['a'])                                                                                                                                                                                                                                                                                                                        
Out[3]: "<a>'example'</a>"
```
Additionally, you ignored my suggestion :) You don't need to initialise the class you just use `clean(value, strip=True, tags=keep)` as in [documentation](https://bleach.readthedocs.io/en/latest/#basic-use)

Reply:
sorry for ignoring the suggestion.
I see that I made a mistake in the example, it has to be without quotes:
<a><b>example</b></a>

Reply:
```
In [4]: xml = "<div><p>example<h1> h1 example</h1></p></div>"
In [5]: clean(xml, strip=True, tags=['p'])
Out[5]: '<p>example\n h1 example</p>'
```